app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
+    if len(parts) == 3:
+        command_name, file, file_copy = parts
```

Comment:
file, file_copy not clear name
first of all it is source_file_path, destination_file_path